prow/github/client.go

```diff
@@ -386,6 +386,24 @@ func (c *Client) CreateStatus(org, repo, ref string, s Status) error {
 	return err
 }
 
+func (c *Client) GetLabels(org, repo string) ([]string, error) {
```

Comment:
This is what we do for the assign plugin. It tries to assign people, but if the return value doesn't have them assigned then it knows they're not an org member.